packages/pypy/build.sh

```diff
@@ -0,0 +1,278 @@
+TERMUX_PKG_HOMEPAGE=https://pypy.org
+TERMUX_PKG_DESCRIPTION="A fast, compliant alternative implementation of Python"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
```

Comment:
Consider setting yourself as maintainer, you know the most about pypy, and you would then also be credited at in the package metadata and repology etc

Reply:
Ok, I see.